src/Development/IDE/LSP/Outline.hs

```diff
@@ -167,6 +167,25 @@ documentSymbolForDecl (L l (ForD x)) = Just
 
 documentSymbolForDecl _ = Nothing
 
+-- | Wrap the Document imports into a hierarchical outline for
+-- a better overview of symbols in scope.
+-- If there are no imports, then no hierarchy will be created.
+documentSymbolForImportSummary :: [DocumentSymbol] -> [DocumentSymbol]
```

Comment:
I think it might be clearer if you make this return a `Maybe DocumentSymbol` and then turn it into a list at the call site. The current type signature doesn’t tell you very much.

Reply:
That is true. However, this requires some minor work at call-site and I thought it would be easier to extend, if we decided to change the behaviour to create import groups per prefix. Will change it, though.